Code/Framework/AzFramework/AzFramework/Spawnable/Script/SpawnableScriptMediator.h

```diff
@@ -66,6 +66,7 @@ namespace AzFramework::Scripts
         {
             EntitySpawnTicket m_spawnTicket;
         };
+        struct Ptr{};
```

Comment:
This can use a more descriptive name such as `CallbackSentinel` or something like that.

Reply:
sounds fancy, I was racking my brain trying to come up with a name hah